cylc/flow/config.py

```diff
@@ -532,6 +532,16 @@ def __init__(
         self._check_special_tasks()  # adds to self.implicit_tasks
         self._check_explicit_cycling()
 
+        for queue in self.cfg["scheduling"]["queues"]:
+            for name in self.cfg["scheduling"]["queues"][queue][
+                "members"
+            ]:
+                if (
+                    name not in self.taskdefs
+                    and name not in self.cfg['runtime']
+                ):
+                    self.implicit_tasks.add(name)
+
```

Comment:
I'm not sure that **failing** for implicit tasks in the queues is a good idea. It is common to have Jinja2 switches which turn bits of graph on and off. This change would mandate matching switches to turn off members in queues too.

```ini
[scheduling]                        
    [[queues]]                      
        [[[myq]]]                   
            members = foo, bar, baz    
    [[graph]]                       
        R1 = """                    
            foo     
            bar     
{% if BAZ %}    
            baz    
{% endif %}    
        """    
     
[runtime]    
    [[foo, bar]]    
{% if BAZ %}    
    [[baz]]    
{% endif %}  
```

There may be value in warning about implicit tasks in queues e.g. to help spot typos.

Maybe bump this bit into a "question" issue for "8.2.0".

Reply:
Agreed, a warning could be helpful, but failing implicit tasks here would create some pain.

Reply:
I thought this was overkill at the moment, but the ticket makes it look like it's a should, not a  "could sometime down the line".